src/main/java/uk/gov/beis/els/categories/waterheaters/controller/WaterHeatersController.java

```diff
@@ -308,6 +310,8 @@ private ModelAndView getWaterSolarPackages(List<FieldError> errorList) {
         WaterHeatersService.LEGISLATION_CATEGORY_SOLAR_PACKAGES);
     modelAndView.addObject("secondaryEfficiencyRating", ControllerUtils.ratingRangeToSelectionMap(
         WaterHeatersService.LEGISLATION_CATEGORY_SOLAR_PACKAGES.getSecondaryRatingRange()));
+    modelAndView.getModel().put("loadProfile", packagesLoadProfiles.stream()
```

Comment:
This can just be `modelAndView.addObject()`

Reply:
@dannybetts wouldn't that throw a duplicate key exception? bc `addCommonObjects()` on line 308 adds the same object

Reply:
never mind xD

Reply:
I thought I would throw a duplicate key exception
Done